pkgs/development/tools/yarn2nix-moretea/yarn2nix/.prettierignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
According to https://prettier.io/docs/en/ignore.html prettier should ignore `node_modules` by default?

Reply:
It's supposed to, but in practice it was running on node_modules without the `.prettierignore`. Perhaps the prettier version is pinned before https://github.com/prettier/prettier/pull/2185 went in, or perhaps it's something in my environment, but it seemed like a safe change.